src/utils/diff/parsers/types.ts

```diff
@@ -1,11 +1,14 @@
+export type MatchKind = 'regular' | 'unknown'
 export type VariableMatch = {
     name: string,
     line: number,
     fileName: string,
-    mode: 'add' | 'remove'
+    mode: 'add' | 'remove',
+    kind: MatchKind
```

Comment:
It isn't obvious what `kind` means, what about having a prop `unidentified` that's true for unidentified matches?